### DISCUSSION TITLE
Redis Job Queue - Reloaded

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#29985] [Quarkus Blog/Website]
# Redis Job Queue - Reloaded

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/redis-job-queue-reloaded/

**Comment by stoyan73** [comment]
Hi Clement, First I want to thank you for the great post! Do you have this project shared in GitHub? I saw the first post but I'm not sure how to implement the changes as you described them here.
